conf/development-websocket.ini

```diff
@@ -1,6 +1,9 @@
 [app:main]
 use: call:h.websocket:create_app
 
+# Elasticsearch settings
+es.host: http://localhost:9200
```

Comment:
I know we talked about leaving these where they were cause they were going to get deleted anyway  but I figured since I was already moving stuff I might as well move this too.